### ISSUE TITLE
Upgrading JFrog products to latest chart version might fail with error "couldn't find key <KEY-NAME> in Secret"

### ISSUE THREAD

**Issue by gitta-jfrog** [jfrog/charts#1819]
**Problem description:** 

Following the release of the latest Helm Charts for JFrog Products,  the following flag is turned on by default:
`unifiedSecretInstallation: true`.

In cases where objects like joinKey and masterKey are defined as custom secrets (joinKeySecretName,masterKeySecretName), application pods might fail to run after upgrading to the latest version with the error:

` Error: couldn't find key <KEY-NAME> in Secret artifactory/artifactory-unified-secret`

**Workaround:**
Disabling unifiedSecretInstallation flag and running the Helm upgrade command.

**artifactory / artifactory-ha**

```
artifactory:
  unifiedSecretInstallation: false
```

**distribution**

```
distribution:
  unifiedSecretInstallation: false
```

**xray**
```
xray:
  unifiedSecretInstallation: false
```

**insight**

```
insightServer:
  unifiedSecretInstallation: false
```

**Comment by chukka** [comment]
Fixed via #1820

**Comment by chukka** [comment]
Please upgrade to  artifactory charts - `107.68.8` and platform chart `10.15.1`
